SPECS/cloud-init/cloud-init.spec

```diff
@@ -109,6 +109,7 @@ echo -e 'line1\nline2\nline3\ncloud-init-ca-certs.crt\n' > "${conf_file}"
 
 pip3 install --upgrade %{test_pkgs}
 pip3 install -r test-requirements.txt
+pip3 install pyserial
```

Comment:
Since we already have the `test_pkgs` macro above why not just add `pyserial` there?